packages/webapp/pages/onboarding.tsx

```diff
@@ -325,7 +325,7 @@ export function OnboardPage(): ReactElement {
               <OnboardingHeadline
                 className={{
                   title: 'tablet:typo-mega-1 typo-large-title',
-                  description: 'typo-body tablet:typo-title2',
+                  description: 'mb-8 typo-body tablet:typo-title2',
```

Comment:
Was this intentional?

Reply:
@sshanzel Yes, mentioned in PR description 😅 

`
I also made slight modifications to OnboardingHeadline so that we pass it the margin we want, and not have to fight with importants as it will be reused.
`

Reply:
Oh, totally missed that 😆